Source/CMakeLists.txt

```diff
@@ -200,11 +201,13 @@ if(WIN32 AND NOT MINGW)
 endif()
 
 # Copy library files to target directory.
-foreach(SDL2_DLL ${WOOF_DLLS})
-    add_custom_command(TARGET woof POST_BUILD
-        COMMAND "${CMAKE_COMMAND}" -E copy_if_different
-        "${SDL2_DLL}" $<TARGET_FILE_DIR:woof> VERBATIM)
-endforeach()
+if(MSVC)
+    foreach(SDL2_DLL ${WOOF_DLLS})
+        add_custom_command(TARGET woof POST_BUILD
+            COMMAND "${CMAKE_COMMAND}" -E copy_if_different
+            "${SDL2_DLL}" $<TARGET_FILE_DIR:woof> VERBATIM)
+    endforeach()
+endif()
```

Comment:
Copying to the target directory (usually woof/build/Source) makes sense only in MSVC. GET_RUNTIME_DEPENDENCIES searches the target directory first, and there might be old library files from libsdl.org.